src/webui/utils/api.js

```diff
@@ -1,34 +1,34 @@
 import storage from './storage';
-import axios from 'axios';
 
 class API {
-  constructor() {
-    ['get', 'delete', 'post', 'put', 'patch'].map((method) => {
-      this[method] = (url, options = {}) => {
-        if (!window.VERDACCIO_API_URL) {
-          throw new Error('VERDACCIO_API_URL is not defined!');
-        }
+  request(url, method = 'GET', options = {}) {
```

Comment:
I'd prefer avoid `['get', 'delete', 'post', 'put', 'patch'].` flow won't be able to go throug it